server/tsuserver.py

```diff
@@ -132,6 +136,10 @@ def start(self):
         if self.config['use_backgrounds_yaml']:
             self.use_backgrounds_yaml = True
 
+        if 'bridgebot' in self.config and self.config['bridgebot']['enabled']:
+            self.bridgebot = Bridgebot(self, self.config['bridgebot']['channel'], self.config['bridgebot']['hub_id'], self.config['bridgebot']['area_id'])
+            asyncio.ensure_future(self.bridgebot.init(self.config['bridgebot']['token']), loop=loop)
```

Comment:
very good, this will run the bridgebot in parallel in the same loop as the main thread